matching-notification-api/src/main/resources/ApplicationResources.properties

```diff
@@ -73,9 +73,9 @@ phenotips.matchingNotifications.table.matchesSeletFilter.outgoing=outgoing match
 phenotips.matchingNotifications.table.matchesSeletFilter.incoming=incoming matches
 
 phenotips.matchingNotifications.table.showMatches.score.label=Matches with score over:
-phenotips.matchingNotifications.table.showMatches.average.label.placeholder=average
-phenotips.matchingNotifications.table.showMatches.phenotipicScore.label.placeholder=phenotypic
-phenotips.matchingNotifications.table.showMatches.genotipicScore.label.placeholder=genotypic
+phenotips.matchingNotifications.table.showMatches.average.label=Avg:
+phenotips.matchingNotifications.table.showMatches.phenotipicScore.label=Phen:
+phenotips.matchingNotifications.table.showMatches.genotipicScore.label=Gen:
```

Comment:
These 3 translation keys are redefined just below, probably to use as slider labels. Please use different translation keys for the filter labels. You could even reuse the table column labels instead:
`phenotips.matchingNotifications.table.headings.averageScore`, `phenotips.matchingNotifications.table.headings.genotypicScore`,
`phenotips.matchingNotifications.table.headings.phenotypicScore`.